WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewController.swift

```diff
@@ -322,18 +360,13 @@ private extension SettingsViewController {
 
     func aboutWasPressed() {
         ServiceLocator.analytics.track(.settingsAboutLinkTapped)
-        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: AboutViewController.self) else {
-            fatalError("Cannot instantiate `AboutViewController` from Dashboard storyboard")
-        }
-        show(viewController, sender: self)
-    }
 
-    func licensesWasPressed() {
-        ServiceLocator.analytics.track(.settingsLicensesLinkTapped)
-        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: LicensesViewController.self) else {
-            fatalError("Cannot instantiate `LicensesViewController` from Dashboard storyboard")
+        let configuration = WooAboutScreenConfiguration()
+        let controller = AutomatticAboutScreen.controller(appInfo: WooAboutScreenConfiguration.appInfo,
+                                                          configuration: configuration)
+        self.present(controller, animated: true) {
+            self.tableView.deselectSelectedRowWithAnimation(true)
```

Comment:
nit: maybe weak self just in case?
```suggestion
        present(controller, animated: true) { [weak self] in
            self?.tableView.deselectSelectedRowWithAnimation(true)
```